### PR TITLE
Fix for issues #3 and #1

### PR DESCRIPTION
**Pull request by narhen**
Throw exceptions on errors, check number of arguments to functions, check type of arguments, fix in _tdControllerValue

### CODE REVIEW

File: telldus-core_node.cc [hunk 1]
```diff
@@ -0,0 +1,1529 @@
+// hello.cc
+#include <iostream>
+#include <node.h>
```
Comment (andreahi) [inline telldus-core_node.cc:3]: Includes should have a logical order

File: telldus-core_node.cc [hunk 2]
```diff
@@ -0,0 +1,1529 @@
...
+			};
+
+			Persistent<Function, CopyablePersistentTraits<Function> > *cb =
+				(Persistent<Function, CopyablePersistentTraits<Function> > *)this->node_callback;
+			Local<Function> callback = Local<Function>::New(isolate, *cb);
+			callback->Call(isolate->GetCurrentContext()->Global(), argc, argv);
+
+			this->free_sensor_strings();
+			uv_mutex_unlock(&async_lock);
+			/*
```
Comment (andreahi) [inline telldus-core_node.cc:175]: remove the comment, if no longer needed

File: telldus-core_node.cc [hunk 3]
```diff
@@ -0,0 +1,1529 @@
+// hello.cc
```
Comment (andreahi) [inline telldus-core_node.cc:1]: There should be an example file, with atleast some basic usage.

**Comment by narhen** [comment]
Does not compile, due to error in binding.gyp

**Pull request closed without merging**